src/test/spec/text-field/text-field.spec.ts

```diff
@@ -237,39 +230,39 @@ describe('TextFieldComponent', function(this: ITestContext) {
       expectFloatingLabelState(this.context, false);
     });
 
-    it('should set dense via property', function(this: ITestContext) {
+    it('should set dense via property', function (this: ITestContext) {
```

Comment:
Do you have a formatter that is updating these files? It may be worth disabling that to reduce noise.

Reply:
I do, and I thought I had nuked all of those before committing, guess I missed some.